srcdoc/configure/access_control.mt

```diff
@@ -0,0 +1,243 @@
+
+<?
+$ctx->{mruby_method}->(
+    name    => "respond",
+    params  => [
+        { label => 'status', desc => 'Status code of the response. Requied.' },
+        { label => 'header', desc => 'Header key-value pairs of the response. Default value: {}' },
+        { label => 'body',   desc => 'Body array of the response. Default value: []' },
+    ],
+    desc    => q{Add a filter which returns arbitrary response if the request matches the provided condition},
+)->(sub {
+?>
+<pre><code>respond(status, header, body) { ..condition.. }</code></pre>
+? })
+
+<?
+$ctx->{mruby_method}->(
+    name    => "use",
+    params  => [
+        { label => 'proc', desc => 'Callable object that should be applied' },
+    ],
+    desc    => q{Add a filter which apply the provided handler (callable object) if the request matches the provided condition},
+)->(sub {
+?>
+<pre><code>use(proc) { ..condition.. }</code></pre>
+? })
+
+<h2 id="matching-methods" class="section-head">Matching Methods</h2>
+
+<p>
+In condition blocks, you can use helpful methods which return particular string values of the request.
+Matching methods can only be used in condition block of the ACL methods.
+</p>
+
+<?
+$ctx->{mruby_method}->(
+    name    => "addr",
+    params  => [
+        { label => 'forwarded', desc => 'If true, use X-Forwarded-For header as the address if it exists. Default value: true' },
+    ],
+    desc    => q{ Returns the remote IP address of the request},
+)->(sub {
+?>
+<pre><code>addr(forwarded)</code></pre>
+? })
+
+<?
+$ctx->{mruby_method}->(
+    name    => "path",
+    desc    => q{ Returns the requested path string of the request},
+)->(sub {
+?>
+<pre><code>path()</code></pre>
+? })
+
+<?
+$ctx->{mruby_method}->(
+    name    => "method",
+    desc    => q{ Returns the HTTP method of the request},
+)->(sub {
+?>
+<pre><code>method()</code></pre>
+? })
+
+<?
+$ctx->{mruby_method}->(
+    name    => "header",
+    params  => [
+        { label => 'name', desc => 'Case-insensitive header name. Requied.' },
+    ],
+    desc    => q{ Returns the header value of the request associated with the provided name},
+)->(sub {
+?>
+<pre><code>header(name)</code></pre>
+? })
+
+<?
+$ctx->{mruby_method}->(
+    name    => "user_agent",
+    desc    => q{ Shortcut for header("user-agent")},
+)->(sub {
+?>
+<pre><code>user_agent()</code></pre>
+? })
+
+<h2 id="caution" class="section-head">Caution</h2>
+
+<p>
+To avoid miss-configuring access control, there are some rules in using <code>acl</code> method.
+<ul>
+<li><code>acl</code> method can be called only once in each handler configuration</li>
+<li>If <code>acl</code> method is used, the evaluation result of the configuration is exactly equals to the return value of <code>acl</code> method</li>
+</ul>
+If a configuration violates these rules, the server will detect it and abort with error message.
+</p>
+
+<p>
+For example, both of the following examples are violating the rules above, so the server will abort.
```

Comment:
s/are violating/violate/
s/the server will abort/the server will refuse to start up/